simcon_project/users/models/assignment.py

```diff
@@ -1,13 +1,16 @@
 from django.db import models
-from conversation_templates.models.conv_template import ConversationTemplate
+import uuid
+
 
 class Assignment(models.Model):
-    id = models.UUIDField(unique=True, editable=False, primary_key=True)
+    id = models.UUIDField(unique=True, editable=False, primary_key=True, default=uuid.uuid4)
     name = models.CharField(max_length=100) 
     date_assigned = models.DateField()
-    templates = models.ForeignKey(ConversationTemplate, related_name='assignments')
-    students = models.ManyToMany(Student, related_name='assignments')
-    labels = models.ManyToMany(SubjectLabel, related_name='assignments')
+    templates = models.ManyToManyField('conversation_templates.ConversationTemplate', related_name='assignments')
```

Comment:
I think this should be called conversation_templates. Its kind of clunky but I think its worth committing to calling these references by the name of the referred model for consistency.

Same goes for subject_labels in this model.